storage/innobase/xtrabackup/src/CMakeLists.txt

```diff
@@ -21,7 +21,10 @@ OPTION(WITH_VERSION_CHECK "Build with version check" ON)
 INCLUDE(${MYSQL_CMAKE_SCRIPT_DIR}/compile_flags.cmake)
 
 FIND_GCRYPT()
-FIND_PROCPS()
+
+IF(NOT APPLE)
+  FIND_PROCPS()
+ENDIF()
```

Comment:
what is necessary for apple, do we need to add apple specific checks?

Reply:
@satya-bodapati 
Actually I tried to explain this in commit message, probably I need to update it :)
`Procps` is used for calculation of memory required for `prepare` (PXB-2710) where  apple uses `sys/sysctl` so nothing else is necessary for apple.
It is just the check to not search for procps in apple was missing in cmake file